### PR TITLE
progress_max parameter added to ProgressReporter

### PR DESCRIPTION
**Pull request by okolo**

### CODE REVIEW

File: oda_api/api.py [hunk 1]
```diff
@@ -1361,16 +1361,19 @@ def __init__(self):
     def enabled(self):
         return self._callback is not None
 
-    def report_progress(self, stage: str=None, progress: int=50, substage: str=None, subprogress: int=None, message:str=None):
+    def report_progress(self, stage: str=None, progress: int=50, progress_max: int=100, substage: str=None,
+                        subprogress: int=None, message:str=None):
```
Comment (dsavchenko) [inline oda_api/api.py:1365]: Probably also `subprogress_max`?

Comment (okolo) [reply oda_api/api.py]: subprogress_max was added in the latest commit

File: oda_api/api.py [hunk 2]
```diff
@@ -1361,16 +1361,19 @@ def __init__(self):
     def enabled(self):
         return self._callback is not None
 
-    def report_progress(self, stage: str=None, progress: int=50, substage: str=None, subprogress: int=None, message:str=None):
+    def report_progress(self, stage: str=None, progress: int=50, progress_max: int=100, substage: str=None,
```
[the remark this answers is not in the document]
Comment (okolo) [reply oda_api/api.py]: progress, subprogress, progress_max and subprogress_max types were changed to float in the latest commit

**Review by dsavchenko — APPROVED** [review]

**Comment by burnout87** [comment]
Anything else to add @okolo ? I would merge it since I need it for https://github.com/oda-hub/mmoda-frontend-module/pull/223

**Review by volodymyrss — APPROVED** [review]

**Comment by okolo** [comment]
> Anything else to add @okolo ? I would merge it since I need it for [oda-hub/mmoda-frontend-module#223](https://github.com/oda-hub/mmoda-frontend-module/pull/223)

@burnout87  I think it's ready. Please merge it! I am not authorized to merge the branch.

**Review by burnout87 — APPROVED** [review]

**Pull request merged**